services/paymaster/src/bin/tx_validator.rs

```diff
@@ -97,14 +98,27 @@ impl Network {
             Network::Testnet => "https://testnet.fogo.io",
         }
     }
+
+    fn default_ntt_quoter_address(self) -> H160 {
+        match self {
+            Network::Mainnet => [
```

Comment:
similar to other fields in the tx validator cli tool, this should not be something users of the tool have to specify. as we have it rn, there will likely be a single quoter per network mode, so users should not need to override